AirCasting/CreateSessionViews/ChooseCustomLocationView.swift

```diff
@@ -29,13 +30,16 @@ struct ChooseCustomLocationView: View {
         }
         .background(confirmCreatingSessionLink)
         .sheet(isPresented: $isLocationPopupPresented) {
-            PlacePicker(address: $location)
+            PlacePicker(placePickerDismissed: $placePickerDismissed, address: $location)
         }
+        .onChange(of: placePickerDismissed, perform: { value in
+            placePickerDismissed ? (placePickerDismissed = false) : nil
```

Comment:
This code is hard to understand, but i don't have a better idea for it - please add a comment here :)